### PR TITLE
2022 Febuary Ports - Part 1

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Part 1 of a large number of upstream ports

## Why It's Good For The Game

Cherrypicked features, fixes, and upgrades from upstream. Split up for easier reviewing (though not fully atomized because haha 900 commits to sift through...)

## Changelog
```changelog
admin: removed references to the Maxim for now
add: hand slowdown variable (6348)
add: Improvised multitool (6370)
add: Generic armor disk (6375)
fix: human movement runtime fix (6358)
fix: Headbutt typo and respect armor (6364)
fix: Ethanol and mead no longer share a recipe. (6377)
fix: Bug with toxins (6365)
sprite: suit sprits from upstream (6369)
```

### CODE REVIEW

File: zzzz_modular_occulus/code/modules/mob/piggyback.dm [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+	set name = "Piggyback"
+	set desc = "climb on someone's back."
+	set category = "IC"
+
+	var/mob/living/carbon/human/target
+	if(!is_physically_disabled())
+		target = get_grabbed_mob(src)
+		if(target)
+			to_chat(src, "CLIMB ON THEY")
+			to_chat(target, "GET CLIMBED ON")
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/modules/mob/piggyback.dm:11]: ?

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]

File: zzzz_modular_occulus/code/modules/mob/piggyback.dm (hunk 1 above)
Comment (Jamini) [reply zzzz_modular_occulus/code/modules/mob/piggyback.dm:11]: How did that get in there.

File: code/game/machinery/excelsior/ex_turret.dm [hunk 2]
```diff
@@ -2,6 +2,7 @@
 #define TURRET_SECONDARY_TARGET 1
 #define TURRET_NOT_TARGET 0
```
Comment (Shadow-Quill) [inline code/game/machinery/excelsior/ex_turret.dm:3]: You'll also want to comment out these defines, else they'll just dangle around elsewhere.

**Review by Shadow-Quill — DISMISSED** [review]

**Comment by Shadow-Quill** [comment]
Also CI failure.

File: code/game/machinery/excelsior/ex_turret.dm (hunk 2 above)
Comment (Jamini) [reply code/game/machinery/excelsior/ex_turret.dm:3]: Tis not needed and not hurting anything.

**Comment by Jamini** [comment]
> Also CI failure.

image
???

**Review by Shadow-Quill — APPROVED** [review]
Strange.

**Pull request merged**